src/main/webapp/app/exercises/modeling/participate/modeling-submission.service.ts

```diff
@@ -24,10 +24,10 @@ export class ModelingSubmissionService {
             .map((res: EntityResponseType) => this.convertResponse(res));
     }
 
-    update(modelingSubmission: ModelingSubmission, exerciseId: number): Observable<EntityResponseType> {
+    update(modelingSubmission: ModelingSubmission, exerciseId: number, submissionId: number): Observable<EntityResponseType> {
```

Comment:
Same here. Isn't the `submissionId` argument redundant here since it can be retrieved from `modelingSubmission`?

Reply:
I agree, let's revert this change!

Reply:
Ok, I reverted this change